package.json

```diff
@@ -14,7 +14,8 @@
     "lint:style": "vue-cli-service lint:style",
     "proxy": "node ./tasks/local-testnet/proxy.js",
     "release": "git checkout develop & git pull & git push origin develop:release",
-    "test:coverage": "live-server test/unit/coverage/lcov-report",
+    "test:coverage": "live-server ./coverage/lcov-report",
+    "test:coverage-branch": "npm run test:unit -- --coverage && git diff origin/develop | diff-test-coverage -c ./coverage/lcov.info -t lcov --",
```

Comment:
allows to test the coverage on your current changes

Reply:
nice!